src/CacheTagsGetterTrait.php

```diff
@@ -37,22 +35,22 @@
      *
      * This method adds the default tags to every tag collection.
      *
-     * @param DynamicEntry|ResourceArray|array|mixed $contentfulResult The result for a contentful query.
+     * @param Entry|ResourceArray|array|mixed $contentfulResult The result for a contentful query.
      *
      * @return array
      */
     protected function getCacheTags($contentfulResult): array
     {
         $tags = $this->getDefaultTags();
 
-        if ($contentfulResult instanceof DynamicEntry) {
+        if ($contentfulResult instanceof Entry) {
             $tags[] = $contentfulResult->getId();
             $contentType = $contentfulResult->getContentType();
             $fields = $contentType->getFields() ?: [];
 
             if ((in_array($contentType->getId(), $this->getRoutableTypes()) &&
                 ($slugFieldName = $this->getSlugField()) &&
-                ($slugField = $contentfulResult->{'get' . ucfirst($slugFieldName)}()))) {
+                ($slugField = $contentfulResult->offsetGet(ucfirst($slugFieldName))))) {
```

Comment:
Please use Array Access, magic property or get, but not the "implicit offset" from the ArrayAccess Interface.

Reply:
I changed it to Array Access.